### PR TITLE
Version 1.0.2

### PR DESCRIPTION
**Pull request by karpetrosyan**
# Changelog

## 1.0.2 

- Fix `float("inf")` timeouts in `Event.wait` function. (#846)

### CODE REVIEW

**Comment by karpetrosyan** [comment]
Do I need to draft a new release? @tomchristie

**Comment by tomchristie** [comment]
Release draft... https://github.com/encode/httpcore/releases/tag/untagged-c331e0a99a76f1769a06

**Comment by tomchristie** [comment]
Looks okay, yup. Shall we release this today?

**Comment by karpetrosyan** [comment]
> Shall we release this today?

Yeah, let's do it

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## Unreleased
+## 1.0.2 (TODO)
```
Comment (tomchristie) [inline CHANGELOG.md:7]:
```suggestion
## 1.0.2 (November 10th, 2023) 
```

**Review by tomchristie — APPROVED** [review]

**Pull request merged**